test/Test/Iris/Cli.hs

```diff
@@ -1,16 +1,21 @@
 {-# LANGUAGE ScopedTypeVariables #-}
 module Test.Iris.Cli (cliSpec) where
 
-import Test.Hspec (Spec, describe, it, shouldBe, expectationFailure)
+import Test.Hspec (Spec, describe, it, shouldBe, expectationFailure, shouldReturn)
 
 import Iris.Cli.ParserInfo (cmdParserInfo)
 import Iris.Settings (defaultCliEnvSettings, cliEnvSettingsVersionSettings)
 import Iris.Cli.Version (defaultVersionSettings)
 import qualified Options.Applicative as Opt
 import qualified Paths_iris as Autogen
 import Iris.Cli (VersionSettings(versionSettingsMkDesc))
+import Iris.Cli.Interactive (handleInteractiveMode, InteractiveMode (..))
+import System.Environment (getEnvironment)
 
 
+checkCI :: IO Bool
+checkCI = (== Just "true") . lookup "CI" <$> getEnvironment
```

Comment:
You can use `lookupEnv` from `System.Environment` instead to simplify the code
```suggestion
checkCI = (== Just "true") <$> lookupEnv "CI"
```

Reply:
will try to find out what it does in ghcup